rules/linux/linux_netcat_network_connection.toml

```
@@ -34,7 +34,7 @@ tags = ["Elastic", "Host", "Linux", "Threat Detection"]
 type = "eql"
 
 query = '''
-sequence by process.entity_id
+sequence by host.name, process.pid with maxspan=30s
```

```suggestion
/* in this instance, process.entity_id is not used in order to be compatible with auditbeat */

sequence by host.id, process.pid with maxspan=30s
```

Also, can you bump the `updated_date`, please